src/spack_commands.py

```diff
@@ -55,13 +70,13 @@ def log_with_spack(command: str,
     # '2>&1' redirects stderr to stdout.
     if env is None:
         ret = subprocess.run(
-            f'{spack_env}; ({srun} {command}) >> {log_file} 2>&1',
+            f'{srun} {uenv_args} bash -c "{spack_env}; {command} >> {log_file} 2>&1" ',
```

Comment:
The brackets in `({srun} {command}) >> {log_file} 2>&1` made this command catch SLURM errors, like timeouts. With this change that's not the case any more. IMHO this is a degradation.

Reply:
Did not realize this makes a difference. Brackets are back in place